starter-code/routes/passportRouter.js

```diff
@@ -1,22 +1,74 @@
-const express        = require("express");
-const router         = express.Router();
+const express = require("express");
+const router = express.Router();
 // User model
-const User           = require("../models/user");
+const User = require("../models/user");
 // Bcrypt to encrypt passwords
-const bcrypt         = require("bcrypt");
-const bcryptSalt     = 10;
+const bcrypt = require("bcrypt");
+const bcryptSalt = 10;
 const ensureLogin = require("connect-ensure-login");
-const passport      = require("passport");
+const passport = require("passport");
 
 
 
 router.get("/private-page", ensureLogin.ensureLoggedIn(), (req, res) => {
-  res.render("passport/private", { user: req.user });
+  res.render("passport/private", {
+    user: req.user
```

Comment:
you can use a global middleware for this instead and set `req.locals.user` to get it on every template